### DISCUSSION TITLE
Extract table which has one row is divided into two pages in PDF

### DISCUSSION THREAD

**Discussion by minhbtrc** [jsvine/pdfplumber#1086] [Ask for help with specific PDFs]
Hello everyone, I'm trying to use PDFplumber to parse my PDF file. Everything seems good, but just one problem that I have been tried to solve but I couldn't fix it.

I have a table, its header row is divided into two pages like this. As you can see, the text of the middle cell is also divided into two pages. I tried many ways from Discussion but still can't fix it. 

![image](https://github.com/jsvine/pdfplumber/assets/58823673/7fdb22db-74c1-4b43-9de1-033faa07e025)

Do you have any idea to help me? Thanks

**Comment by jsvine** [comment]
Hi @spartan-minhbui, and thanks for your interest in this library. PDFs are inherently page-based, and so there aren't many ways to easily handle tables that cross two pages. When I've dealt with this issue in the past, I've ended up writing custom code — but the code will depend substantially on the particular PDF and the ways in which you expect the tables to be divided. 

If the splitting is predictable enough, something like the `explicit_horizontal_lines` table-extraction setting can be useful; you'd pass in the y-position of where the bottom line (in the first page) and top line (in the second) would be. Then you could extract each table from each page, and join them later in the process.
